extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -347,7 +348,8 @@ export const targetPlatformToVersion: Map<string, string> = new Map<string, stri
 	[sqlServer2016, '130'],
 	[sqlServer2017, '140'],
 	[sqlServer2019, '150'],
-	[sqlAzure, 'AzureV12']
+	[sqlAzure, 'AzureV12'],
+	[sqlDW, 'Dw']
```

Comment:
small case w expected?

Reply:
Yup. SSDT is strict on the casing of the version. Only Dw is allowed - DW, dW, and dw aren't allowed